### PR TITLE
Removed zones from cockroach demo

### PR DESCRIPTION
**Pull request by Zhaars**
Removed zones usage for the demo with cockroachDB

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,20 +5,20 @@ Acra Engineering Examples illustrate the integration of [Acra data protection su
...
+| 6  | [Client-side encryption with Zones, python app, PostgreSQL (deprecated, will be removed)](https://github.com/cossacklabs/acra-engineering-demo/#example-6-client-side-encryption-zones-python-app-postgresql) | Simple python client application, client-side encryption with Zones support, decryption on AcraServer, PostgreSQL |
+| 7  | [Client-side encryption, Ruby on Rails app, PostgreSQL](https://github.com/cossacklabs/acra-engineering-demo/#example-7-client-side-encryption-ruby-on-rails-app-postgresql)                                  | Ruby on Rails web application, client-side encryption, decryption on AcraServer, PostgreSQL                       |
+| 8  | [Transparent encryption, Zones, python app, CockroachDB](https://github.com/cossacklabs/acra-engineering-demo/#example-8-transparent-encryption-zones-python-app-cockroachdb)                                 | Simple python client application, transparent encryption/decryption on AcraServer, CockroachDB                    |
```
Comment (vixentael) [inline README.md:17]: as we will rename the example, the link will be changed

File: README.md [hunk 2]
```diff
@@ -963,53 +963,23 @@ database, sets up the environment, configures python application to connect to A
...
-
-### 2.3 Restart acra-server to use updated config
-
-```bash
-docker restart cockroachdb_acra-server_1
-```
-
-
-### 2.4 Insert data using updated config
+### 2.4 Insert data
```
Comment (vixentael) [inline README.md:967]: we should fix numbers (enumeration) after removing some of the items.

```suggestion
### 2.1 Insert data
```

**Review by Lagovas — APPROVED** [review]
lgtm

**Review by vixentael — APPROVED** [review]
:dancing_dog:

**Pull request merged**